library/macos/FLETextInputModel.m

```diff
@@ -70,9 +74,21 @@ static long GetExtentForRange(NSRange range) {
 
 @implementation FLETextInputModel
 
-- (instancetype)init {
+- (nullable instancetype)initWithClientID:(nonnull NSNumber *)clientID
```

Comment:
Please remove these three nullability annotations; we don't use them on implementations in this project. (With Xcode not enforcing that they match, it's easy for them to be wrong.)